### PR TITLE
Spike: add lighthouse-ci workflow

### PR DESCRIPTION
**Pull request by andrewHEguardian**
## What are you doing in this PR?

Part of a Spike on how we might measure page speed with reports, and alert against budgets. This PR implements https://github.com/treosh/lighthouse-ci-action to monitor our prod site after every release to assert against a set of performance budget.

[**Trello Card**](https://trello.com/c/rZDwH4l3/1102-spike-page-speed-github-actions)

When the assertions fail we can see the result in the Github action and 

image

in the assertions.json file in the artifact 
image

### CODE REVIEW

File: .github/workflows/lighthouse.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  Lighthouse:
+    if: ${{ github.event.label.name == 'Seen-on-PROD' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Audit URLs using Lighthouse
+        uses: treosh/lighthouse-ci-action@v12
+        with:
+          urls: |
+            https://support.theguardian.com/uk/contribute
```
Comment (rupertbates) [inline .github/workflows/lighthouse.yml:16]: We should have a think about other urls to add here. The subscribe ones for instance.

File: lighthouserc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "ci": {
+    "assert": {
+      "assertions": {
```
Comment (rupertbates) [inline lighthouserc.json:4]: How did you decide on these assertions?

**Review by rupertbates — APPROVED** [review]
Great, really nice and simple 👍

File: lighthouserc.json (hunk 2 above)
Comment (andrewHEguardian) [reply lighthouserc.json:4]: Some were in examples and some are prominent in the lighthouse report. But we can definitely tweak it to what we want
image

**Review by paul-daniel-dempsey — COMMENTED** [review]
Like this, great job finding out. A lot of settings ned to get familiar with I guess..

**Pull request merged**